tests/storage/hotplug.go

```diff
@@ -1385,9 +1383,9 @@ var _ = SIGDescribe("Hotplug", func() {
 
 		BeforeEach(func() {
 			exists := false
-			sc, exists = libstorage.GetRWOFileSystemStorageClass()
+			sc, exists = libstorage.GetRWXBlockStorageClass()
```

Comment:
Switch to block? why so?

Reply:
Sorry I thought commit msg was empty. ack

Reply:
Because we have a bind immediate on the volume, and the volume could end up on the wrong and then hotplugging will fail because it cannot start the pod due to the VM on node1 and the volume on node2. Making it block forces the use of ceph in CI, and thus no issues on that front. Since for this test we are interested in the pod req/limit ratio and not storage, I decided to switch back to block.